packages/cadl-rlc-emitter/test/commands/cadl-ranch-list.ts

```diff
@@ -32,18 +32,6 @@ export const cadls: CadlRanchConfig[] = [
     outputPath: "models/inheritance",
     inputPath: "models/inheritance"
   },
-  {
-    outputPath: "models/inputBasic",
-    inputPath: "models/input-basic"
-  },
-  {
-    outputPath: "models/nestedModels",
-    inputPath: "models/nested-models"
-  },
-  {
-    outputPath: "models/outputBasic",
-    inputPath: "models/output-basic"
-  },
```

Comment:
Are they removed because of latest cadl-ranch ?

Reply:
yes